pkg/daemon/containerrecreate/crr_daemon_util.go

```diff
@@ -128,6 +136,16 @@ func getCurrentCRRContainersRecreateStates(
 	return statuses
 }
 
+func containerIsNotForceKilled(recreateState *appsv1alpha1.ContainerRecreateRequestContainerRecreateState) bool {
```

Comment:
containerIsNotForceKilled  -> containerIsForceKilled  
plz revert the func logic to make code more easy to understand